public/elmish/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv='Content-Type' content='text/html; charset=utf-8'>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,fetch"></script>  
```

Comment:
You can remove this scrip tag because we now include `core-js` by default. And we don't need fetch :)

Reply:
Done 👍